sdk/elixir/lib/dagger/core/error.ex

```diff
@@ -20,36 +20,6 @@ defmodule Dagger.Core.ExecError do
 
   @impl true
   def message(exception) do
-    stdout =
-      if String.trim(exception.stdout) != "" do
-        [
-          "Stdout:",
-          ?\n,
-          exception.stdout
-        ]
-      else
-        ""
-      end
-
-    stderr =
-      if String.trim(exception.stderr) != "" do
-        [
-          "Stderr:",
-          ?\n,
-          exception.stderr
-        ]
-      else
-        ""
-      end
```

Comment:
So we can remove `cmd`, `stdout`, `stderr` and `exit_code` out of the exception?

Reply:
I would keep them - that way users can still check for them if they're able to assert against the concrete error type. We went through the same for Go: initially disabled collecting that info in entirely #8440, but with this PR we brought it back, kept the fields, and just removed them from the display method (`Error() string`).